bugbug/similarity.py

```diff
@@ -349,3 +349,30 @@ def get_similar_bugs(self, query):
             for similar in sorted(similarities, key=lambda v: v[1])[:10]
             if similar[0] != query["id"] and similar[1] < self.cut_off
         ]
+
+    def get_distance(self, query1, query2):
```

Comment:
It might be worth adding this function for all Similarity models, but for now you could just define an abstract function for in the BaseSimilarity class and then we'll implement it for the other models later.